examples/raspberry-pi/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "raspberry-pi"
```

Comment:
It can be done as examples/raspberrypi.rs, but need add some .cargo settings to library folder(such linkers and other settings). 
Should I untrack this project inside ? (I'm using to debug/test, since the driver is agnostic)

Reply:
It's possible, but will need add unnecessary settings to main library, It's was being used to test/debug the lib, since the drivers are agnostic made. 

Should untrack the project from library?

Reply:
if we move this code to an example we can remove mostly of the files here.